moto/elasticache/exceptions.py

```diff
@@ -99,6 +99,28 @@ def __init__(self, cache_cluster_id: str):
         )
 
 
+class CacheSubnetGroupAlreadyExists(ElastiCacheException):
+    code = 404
+
+    def __init__(self, cache_subnet_group_name: str):
+        (
+            super().__init__(
+                "CacheSubnetGroupAlreadyExists",
+                message=f"CacheSubnetGroup {cache_subnet_group_name} already exists.",
+            ),
+        )
```

Comment:
```suggestion
        super().__init__(
            "CacheSubnetGroupAlreadyExists",
            message=f"CacheSubnetGroup {cache_subnet_group_name} already exists.",
        )
```
A nitpick, but this broke my brain a bit trying to understand what's going on :slightly_smiling_face:

Reply:
Ah, I copied the existing CacheClusterAlreadyExists exception and it did it that way. Will remove it from there too.